src/decaylanguage/decay/decay.py

```diff
@@ -602,6 +602,33 @@ def ndecays(self) -> int:
         """
         return len(self.decays)
 
+    def to_string(self, arrow="->") -> str:
+        """
+        One-line string representation of the entire decay chain. Sub-decays are
+        enclosed in round parentheses.
+        """
```

Comment:
It would be nice to add your example
```
>>> dm1 = DecayMode(0.6770, "D0 pi+") # D*+
>>> dm2 = DecayMode(0.0124, "K_S0 pi0") # D0
>>> dm3 = DecayMode(0.692, "pi+ pi-") # K_S0
>>> dm4 = DecayMode(0.98823, "gamma gamma") # pi-
>>> dc = DecayChain("D*+", {"D*+":dm1, "D0":dm2, "K_S0":dm3, "pi0":dm4})
>>> print(dc.to_string())
D*+ -> (D0 -> (K_S0 -> pi+ pi-) (pi0 -> gamma gamma)) pi+
```
in the docstring as I tend to do in dedicated "sections".